src/Modal/Modal.test.jsx

```diff
@@ -221,38 +221,11 @@ describe('<Modal />', () => {
       wrapper.find('button').at(0).simulate('click');
       modalOpen(false, wrapper);
       wrapper.setProps({ open: true });
+      wrapper.update();
       modalOpen(true, wrapper);
     });
   });
-  describe('invalid keystrokes do nothing', () => {
```

Comment:
now that the `Modal` uses [react-focus-on](https://www.npmjs.com/package/react-focus-on) (and react-focus-lock under the hood) in favor of custom keydown events, these tests are no longer relevant.